mgmt_cli_test.py

```diff
@@ -251,13 +250,7 @@ def generate_background_read_load(self):
         number_of_nodes = self.params.get("n_db_nodes")
         number_of_loaders = self.params.get("n_loaders")
 
-        scylla_version = self.db_cluster.nodes[0].scylla_version
-        if parse_version(scylla_version).release[0] == 2019:
-            # Making sure scylla version is 2019.1.x
-            throttle_per_node = 10666
-        else:
-            throttle_per_node = 14666
-
+        throttle_per_node = 14666
```

Comment:
Just out of the curiosity. What is the meaning of this value ?

Reply:
Actually, I have no clue where it comes from originally and why it's exactly 14666.
In this PR I've only removed the separate logic for Enterprise release 2019 handling.

Probably @ShlomiBalalis can help us with that?

Reply:
probably this number must be revalidated - Scylla performance and hardware underneath changed.